package.json

```diff
@@ -4,14 +4,64 @@
   "version": "0.0.1",
   "description": "The addon where ideas come to idea",
   "main": "index.js",
-  "author": "Les Orchard <lorchard@mozilla.com>",
+  "author": "Mozilla (https://mozilla.org/)",
   "engines": {
     "firefox": ">=38.0a1",
     "fennec": ">=38.0a1"
   },
-  "license": "MIT",
+  "scripts": {
+    "start": "st -nc .",
+    "lint": "eslint",
```

Comment:
Running `npm run lint` doesn't exactly _do_ anything (it only brings up the usage/help for me locally).

You'll need to change it to "eslint ." and then we can manage ignored files/folders using an .eslintignore file.